docs/upgrade_history/v27_evm_emulation/v27-evm-emulation.md

```diff
@@ -0,0 +1,56 @@
+# V27 EVM emulation upgrade
+
+## Upgrade process
+
+V27 upgrade will happen after the gateway preperation upgrade, but before the gateway is deployed. As such the ugprade process does not involve the Gateway parts ( upgrading the CTM on GW, pausing migrations, etc), it is an L1-only upgrade. 
```

Comment:
preperation, ugprade

Reply:
the comma is not needed I think. It is the gateway preparation upgrade, i.e. we prepare for GW.